specification/ai/Azure.AI.Projects/notifications/models.tsp

```diff
@@ -0,0 +1,104 @@
+import "@typespec/rest";
+import "@azure-tools/typespec-autorest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "../common/models.tsp";
+import "../main.tsp";
+import "@typespec/openapi";
+import "@typespec/versioning";
+
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+
+namespace Azure.AI.Projects;
+
+@doc("Notification title information")
+@added(Versions.v2025_05_15_preview)
+model NotificationTitle {
```

Comment:
nit: since this contains multiple fields, maybe name it NotificationInformation/Label/something that makes it clear that it's not just a title